tests/hmm/analysis_hmumu.py

```diff
@@ -335,17 +335,19 @@ def __init__(self, args, do_tensorflow=True, gpu_memory_fraction=0.2):
             if args.use_cuda:
                 import cupy
                 self.dnn_normfactors = cupy.array(self.dnn_normfactors[0]), cupy.array(self.dnn_normfactors[1])
-
-            json_path = "data/PisaDNN/model_preparation/model_toexport_evt0.json"
-            with open(json_path, 'r') as file_handle:
-                self.dnnPisa_model = keras.models.model_from_json(file_handle.read())
-            self.dnnPisa_model.load_weights("data/PisaDNN/model_preparation/model_toexport_evt0.h5")
-            self.dnnPisa_normfactors1 = np.load("data/PisaDNN/model_preparation/helphelp_node1.npy")
-            self.dnnPisa_normfactors2 = np.load("data/PisaDNN/model_preparation/helphelp_node2.npy")
-            if args.use_cuda:
-                import cupy
-                self.dnnPisa_normfactors1 = cupy.array(self.dnnPisa_normfactors1[0]), cupy.array(self.dnnPisa_normfactors1[1])
-                self.dnnPisa_normfactors2 = cupy.array(self.dnnPisa_normfactors2[0]), cupy.array(self.dnnPisa_normfactors2[1])
+            
+            for imodel in range(4):
```

Comment:
to make maintaining the code easier in the future, I would suggest to take this `4` from the parameters dictionary, as it is needed also later in the code.